src/components/Post/PostContent.tsx

```diff
@@ -0,0 +1,150 @@
+import { useNavigate } from 'react-router-dom';
+import styled from 'styled-components';
+import { COLOR } from '../../utils/color';
+import { queryLowImage } from '../../utils/image';
+import { ROUTES } from '../../utils/routes';
+import Image from '../Base/Image';
+
+interface PostContentProps {
+  _id: string;
+  title: string;
+  image?: string;
+  isDetailPage?: boolean;
+  isMyLikes?: boolean;
+}
+
+interface PostDetailProps {
+  isDetailPage?: boolean;
+  isMyLikes?: boolean;
+}
+
+interface ImageContainerProps {
+  width?: string;
+  height?: string;
+  isMyLikes?: boolean;
+}
+
+const PostContent = ({
+  _id,
+  title,
+  image,
+  isDetailPage,
+  isMyLikes,
+}: PostContentProps) => {
+  const navigate = useNavigate();
+  const postContent = title[0] === '{' ? JSON.parse(title) : title;
```

Comment:
```suggestion
  const postContent = JSON.parse(title);
```
이 부분은 처음에 우리가 테스트용으로 데이터 직접 넣어줬을 때 string으로 넣어줘서 일단 처리해줬던 부분인데, 이제는 서버 데이터만 가져오니까 제안한 코드처럼 해줘도 될 것 같다는 생각이 들어요! 어떻게 생각하시나요?ㅎㅎ

Reply:
이제는 제목과 바디 둘다 있어야만 가능하니까 수정해도 좋을것 같습니다 반영해서 올리겠습니다~